architecture/index.adoc

```diff
@@ -103,3 +103,5 @@ endif::openshift-dedicated,openshift-rosa[]
 [id="about-admission-plug-ins"]
 == About admission plugins
 You can use xref:../architecture/admission-plug-ins.adoc#admission-plug-ins[admission plugins] to regulate how {product-title} functions. After a resource request is authenticated and authorized, admission plugins intercept the resource request to the master API to validate resource requests and to ensure that scaling policies are adhered to. Admission plugins are used to enforce security policies, resource limitations, configuration requirements, and other settings.
+
+include::modules/architecture-about-cgroup-v2.adoc[leveloffset=+2]
```

Comment:
Hi, I was wondering if this is supposed to be a heading *under* admission plugins, or on the same level/hierarchy? 

![Screenshot 2025-05-05 at 4 28 34 PM](https://github.com/user-attachments/assets/cd154bc7-feac-4e10-9ffa-d262654d996b)